### DISCUSSION TITLE
iframes are not scrolling in carousel

### DISCUSSION THREAD

**Discussion by liorPiggy** [nolimits4web/swiper#6006] [Q&A]
this is how iframes work

_Originally posted by @nolimits4web in https://github.com/nolimits4web/swiper/issues/5980#issuecomment-1225349026_

since this was closed before i had the chance to respond or explain, and on top of that the response was just redundant.
iv'e checked iframes outside of the carousel and do enable scrolling of the original page.

**Comment by liorPiggy** [comment]
Hey @nolimits4web,
Thanks for opening this as a discussion.
You check this jsbin demo, using an iframe embed copied from youtube, outside of the carousel the mouse scroll event bubbles up to the original document.
I think the root cause of the issue within the carousel is coming from the fact that the event is being caught in the carousel handlers and does not bubble outside of it.

see the demo here:
https://jsbin.com/vekirebuve

**Reply by vltansky** [reply]
https://bugzilla.mozilla.org/show_bug.cgi?id=1084121

http://dtapuska.github.io/564042/iframe-scroll.html
http://jsfiddle.net/8cj0dofx/2/
